lib/trento/sap_systems.ex

```diff
@@ -104,6 +112,40 @@ defmodule Trento.SapSystems do
 
   def request_instance_operation(_, _, _, _), do: {:error, :operation_not_found}
 
+  @spec request_operation(atom(), Ecto.UUID.t(), map()) ::
+          {:ok, String.t()} | {:error, any}
+  def request_operation(operation, sap_system_id, params)
+      when operation in SapSystemOperations.values() do
+    operation_id = UUID.uuid4()
+
+    # Look for 1st running host to send the operation including the instance_number
+    # Not checking if the SAP system is deregistered. That must be done by the function user
+    agents =
```

Comment:
I'm wondering if I put the "policy" to have at least one running host.
Because I want to forbid the operation if this happens.
The other option is to return 404, but it doesn't fully represent the reality.
Thought?

Reply:
Perhaps I just send the first available host otherwise (if no running host if found). Just to not change how we send requests.
It would timeout in the worst case.
At the end, we do the same with any other operation.

Reply:
Alright. In that case what do you think about just removing the `without targets` related test and add a comment in the function implementation?